typescript/cli/src/deploy/core.ts

```diff
@@ -81,6 +82,21 @@ export async function runCoreDeploy(params: DeployParams) {
 
   const userAddress = await signer.getAddress();
 
+  const { technicalStack: chainTechnicalStack } =
+    context.multiProvider.getChainMetadata(chain);
+
+  const ismType =
+    typeof config.defaultIsm === 'string'
+      ? (config.defaultIsm as IsmType)
```

Comment:
@paulbalaji In case config.defaultIsm is of type string, what could the potential value of that type be that would fall outside IsmType enum?